updating/understanding-upgrade-channels-release.adoc

```diff
@@ -6,7 +6,17 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Update channels are tied to a minor version of {product-title}. For instance, {product-title} 4.10 update channels recommend updates to 4.10 and updates within 4.10. They also recommend updates within 4.9 and from 4.9 to 4.10 allowing all 4.9 to eventually update to 4.10, even if they do not immediately meet the minimum z-stream version requirements. They do not recommend updates to 4.11 or later releases. This strategy ensures that administrators explicitly decide to update to the next minor version of {product-title}.
+Update channels are the mechanism by which users declare the {product-title} minor version they intend to update to. The Cluster Version Operator uses an update graph based on the channel declaration, along with other conditional information, to provide a list of recommended updates available to the cluster.
```

Comment:
> Update channels are the mechanism by which users declare the {product-title} minor version they intend to update to.

And also the timing/supported-ness of the recommended updates, e.g. `candidate` vs. `fast` vs. `stable`/`eus`.

Reply:
Thank you, I'll add this info somewhere in the paragraph